frontend/src/routes/Routes.tsx

```diff
@@ -35,14 +35,17 @@ export enum PATHS {
   VIEW_EVENT_ID = '/schedule/events/view/:eventId',
   // FACILITY BOOKING
   FACILITY_BOOKING_MAIN = '/facility',
-  VIEW_FACILITY = '/facility/view',
+  VIEW_FACILITY = '/facility/view/',
   VIEW_FACILITY_ID = '/facility/view/:facilityID',
-  CREATE_FACILITY_BOOKING = '/facility/booking/create',
-  CREATE_FACILITY_BOOKING_ID = '/facility/booking/create/:facilityId',
+  CREATE_FACILITY_BOOKING = '/facility/booking/create/',
+  CREATE_FACILITY_BOOKING_DAILY_VIEW = '/facility/booking/dayview/create/',
```

Comment:
these should not have a trailing /; also facilityID should be facilityId instead, following camelCase